genutil/common.go

```diff
@@ -104,15 +104,19 @@ type CompressBehaviour int64
 // No dimension spans across all others' options, so can't extract any out
 // without having to error out for some combinations.
 const (
-	// Uncompressed means to not compress the schema.
+	// Uncompressed means to not compress the generated code relative to
+	// the schema.
 	Uncompressed CompressBehaviour = iota
-	// UncompressedExcludeDerivedState excludes config false subtrees.
+	// UncompressedExcludeDerivedState excludes config false subtrees in
+	// the generated code.
 	UncompressedExcludeDerivedState
-	// PreferIntendedConfig indicates to use the "config" version of a
-	// schema entry instead of its "state" counterpart when both exist.
+	// PreferIntendedConfig indicates to generate only the "config" version
+	// of a field when it exists under both "config" and "state" containers
+	// of its parent YANG model.
 	PreferIntendedConfig
-	// PreferOperationalState indicates to use the "state" version of a
-	// schema entry instead of its "config" counterpart when both exist.
+	// PreferOperationalState indicates to generate only the "state"
```

Comment:
same as above: "indicates to generate" -> "generates" and what happens when a field exists under one of either "config" or "state"